openedx/features/course_duration_limits/tests/test_course_expiration.py

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
from a quick review these tests look good. once the codecov runs I would just make sure we have 100% coverage on the get_user_course_expiration_date function (which I think you probably already do) since it is particularly important